### ISSUE TITLE
As forgot my watch Dude, I want to be able to reference the time and gauge the arrival times and know how that will fit with my schedule.

### ISSUE THREAD

**Issue by brock8503** [project-monolith/monolith-frontend#29]
We need to explore the idea of time in the design. @jhammerquist I think one point of confusion right now is the lack of time as a point of reference for the tiles. Did we consider a clock at all?

**Comment by eldang** [comment]
This is a good point, which has also come up in conversation for a secondary reason: to anyone who does know the current time a clock readout can make it usefully obvious if the kiosk has frozen and shouldn't be trusted.

**Comment by jhammerquist** [comment]
I agree. Important feature. What do you guys think about this design: http://cl.ly/image/0K2M0V2b2r3o
